src/sentry/replays/query.py

```diff
@@ -184,12 +184,23 @@ def query_replays_dataset_tagkey_values(
     end: datetime,
     environment: str | None,
     tag_key: str,
+    tag_like: str | None,
     tenant_ids: dict[str, Any] | None,
 ):
     """Query replay tagkey values. Like our other tag functionality, aggregates do not work here."""
-
-    where = []
-
+    """This function is used by the tagstore backend, which expects a `tag_value` key in each result object."""
+
+    where = [
+        Condition(Column("project_id"), Op.IN, project_ids),
+        Condition(Column("timestamp"), Op.LT, end),
+        Condition(Column("timestamp"), Op.GTE, start),
+        Or(
+            [
+                Condition(Column("is_archived"), Op.EQ, 0),
+                Condition(Column("is_archived"), Op.IS_NULL),
+            ]
+        ),
+    ]
```

Comment:
Its fine to move this here but it was in the place it was to keep it local to the execution site.  Less scrolling.  You don't have to change this I'm just giving the reasoning.  I defer to your judgement.

Reply:
Got it, thanks for explaining. I moved it up pretty arbitrarily when coding, don't mind moving it back